calico-cloud/get-started/system-requirements.mdx

```diff
@@ -1,62 +1,50 @@
 
-### Step 2: Verify cluster is not managed by a Kubernetes reconciler
+| Distribution | Supported CNIs |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
+| Amazon Elastic Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Amazon VPC CNI |
+| Azure Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Azure CNI |
+| Google Kubernetes Engine | - Calico Open Source 3.26 and 3.27<br/> - GKE CNI |
+| Rancher Kubernetes Engine 2 | - Calico Open Source 3.26 and 3.27 |
 
-{{prodname}} cannot be installed on clusters that are managed by any kind of Kubernetes reconciler (for example, Addon-manager). To verify, look for an annotation called `addonmanager.kubernetes.io/mode` on either of the following resources. (The resources may not exist).
 
-- `tigera-operator` deployment in the `tigera-operator` namespace (Installation on AKS may still work if the mode is Reconcile, see [AKS](aks.mdx#verify-addonmanager-is-compatible) for an additional check.)
-- `calico-node` daemonset in the `kube-system` namespace
+## Kubernetes versions
 
-If the following command finds addonmanager on either of the resources, then Addon-manager is being used. Find a different cluster to use.
+Your Kubernetes distribution must be based on one of the following Kubernetes versions:
+* Kubernetes 1.29
+* Kubernetes 1.28
+* Kubernetes 1.27
+* Kubernetes 1.26
 
-```bash
-kubectl get <type> -n <namespace> <resource-name> -o yaml | grep ' addonmanager.kubernetes.io/mode:'
-```
+## Architectures
 
-### Step 3: Prepare your infrastructure to connect to {{prodname}}
+Calico Cloud can be installed on nodes based on the following chip architectures:
+* x86-64
+* ARM64
 
-Pods running in your Kubernetes cluster must allow outbound traffic to the following endpoints:
+## Browser support for the Manager UI web console
 
-- `https://installer.calicocloud.io:443/*`
-- `https://www.calicocloud.io:443/api/*`
-- `https://client-auth.calicocloud.io:443/*`
-- TCP to `<management-cluster>.calicocloud.io:9000`
+To access the Manager UI web console, you can use latest two versions of the following web browsers:
+* Chrome
+* Firefox
+* Safari
 
-For each node, Docker must be able to pull images from the following endpoints:
+## Next steps
 
-* `quay.io`
-* `cdn01.quay.io`
-* `cdn02.quay.io`
-* `us-docker.pkg.dev`
-
-These connections must be allowed by your Kubernetes cluster and also by the rest of your infrastructure.
-You won't be able to use {{prodname}} if other elements of your infrastructure, such as firewalls or security groups, restrict access to these endpoints.
-
-### Step 4: Verify platform-specific requirements
-
-If you using one of the following platforms, review further requirements. Otherwise, you are ready to connect your cluster in Manager UI: go to Managed clusters, and click **Connect**.
-
-- [AKS](aks.mdx)
-- [GKE](gke.mdx)
-- [RKE2](rke2.mdx)
+* [Prepare your cluster for Calico Cloud](prepare-cluster.mdx)
```

Comment:
You've put many of the requirements under "Prepare your cluster for Calico Cloud", the title suggests if you've made it that far you can connect your cluster you might just need to make some changes which is not the case.

I think all of the items that are now in this new section that were previously in "Verify platform-specific requirements" are all things that in the majority of the clusters cannot be modified.